src/main/java/com/gooddata/featureflag/FeatureFlags.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2007-2015, GoodData(R) Corporation. All rights reserved.
+ */
+package com.gooddata.featureflag;
+
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import org.springframework.web.util.UriTemplate;
+
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+
+import static com.gooddata.util.Validate.notEmpty;
+import static org.springframework.util.Assert.notNull;
+
+@JsonTypeInfo(include = JsonTypeInfo.As.WRAPPER_OBJECT, use = JsonTypeInfo.Id.NAME)
+@JsonTypeName("featureFlags")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class FeatureFlags implements Iterable<FeatureFlag> {
```

Comment:
iterable is cool, are we sure we don't want the real collection? (similarly we have PageableCollection)
i guess we had this conversation before, don't you remember my opinion? :)